### PR TITLE
Dark Mode Updates

### PR DESCRIPTION
**Pull request by bwiggs**
Taking the lights down.

CleanShot 2024-12-03 at 12 59 02
CleanShot 2024-12-03 at 13 01 08

### CODE REVIEW

File: packages/tailwind-config/tailwind.config.ts [hunk 1]
```diff
@@ -92,31 +120,44 @@ export const config: Omit<Config, 'content'> = {
...
+          '50': '#f2f7f9',
+          '100': '#dfeaee',
+          '200': '#c2d7df',
+          '300': '#98bac8',
+          '400': '#6694aa',
+          '500': '#4b788f',
+          '600': '#416379',
+          '700': '#395365',
+          '800': '#354755',
+          '900': '#303e4a', // background color in dark mode
```
Comment (golanglemonade) [inline packages/tailwind-config/tailwind.config.ts:134]: these comments are no longer correct, right? Can you update it to the ones that are the background/font colors for dark mode?

File: apps/console/src/components/pages/protected/questionnaire/theme-dark.tsx [hunk 2]
```diff
@@ -1,175 +1,971 @@
...
+  "--sjs-primary-backcolor-light": "rgba(0, 255, 0, 0.07)",
+  "--sjs-primary-forecolor": "rgba(32, 32, 32, 1)",
+  "--sjs-primary-forecolor-light": "rgba(32, 32, 32, 0.25)",
+  "--sjs-special-red": "rgba(254, 76, 108, 1)",
+  "--sjs-special-red-light": "rgba(254, 76, 108, 0.1)",
+  "--sjs-questionpanel-hovercolor": colors.backgroundDim,
+};
+
+// this theme is from the suveyjs creator colors
+// the ddefault theme variables are here: https://github.com/surveyjs/survey-creator/blob/master/packages/survey-creator-core/src/themes/default.ts
```
Comment (golanglemonade) [inline apps/console/src/components/pages/protected/questionnaire/theme-dark.tsx:82]:
```suggestion
// the default theme variables are here: https://github.com/surveyjs/survey-creator/blob/master/packages/survey-creator-core/src/themes/default.ts
```

**Review by golanglemonade — COMMENTED** [review]
LGTM, thanks!

File: packages/tailwind-config/tailwind.config.ts (hunk 1 above)
Comment (bwiggs) [reply packages/tailwind-config/tailwind.config.ts]: good catch. I've updated the comments.

I also sorted the colors in the list so they group appropriately in storybook. 

We've got some pretty similar colors going on here too, zigg/oxford and neutral/riverbed are pretty close to one another.

CleanShot 2024-12-04 at 11 44 59

**Review by golanglemonade — APPROVED** [review]

**Pull request merged**